current/standard-derivatives.md

```diff
@@ -0,0 +1,552 @@
+warcs
+  .pdfs()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("/path/to/derivatives/csv/pdf")
+
+// Presentation Program Files.
+warcs
+  .presentationProgramFiles()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("/path/to/derivatives/csv/presentation-program")
+
+// Spreadsheets.
+warcs
+  .spreadsheets()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("/path/to/derivatives/csv/spreadsheet")
+
+// Text Files.
+warcs
+  .textFiles()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("/path/to/derivatives/csv/text")
+
+// Videos.
+warcs
+  .videos()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("/path/to/derivatives/csv/video")
+
+// Word Processor Files.
+warcs
+  .wordProcessorFiles()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("/path/to/derivatives/csv/word-processor")
+
+sys.exit
+```
+
+### Python DF
+
+```python
+from aut import *
+
+# Web archive collection (dataset).
+warcs = WebArchive(sc, sqlContext, "/path/to/aut-resources-master/Sample-Data/*gz")
+
+# Audio Files.
+warcs.audio().write.csv('/path/to/derivatives/csv/audio', header='true')
+
+# Images.
+warcs.images().write.csv('/path/to/derivatives/csv/images', header='true')
+
+# Image Links.
+warcs.image_links().write.csv('/path/to/derivatives/csv/images-links', header='true')
+
+# PDFs.
+warcs.pdfs().write.csv('/path/to/derivatives/csv/pdfs', header='true')
+
+# Spreadsheets.
+warcs.spreadsheets().write.csv('/path/to/derivatives/csv/spreadsheets', header='true')
+
+# Presentation Program Files.
+warcs.presentation_program().write.csv('/path/to/derivatives/csv/presentation_program', header='true')
+
+# Text Files.
+warcs.text_files().write.csv('/path/to/derivatives/csv/text_files', header='true')
+
+# Videos.
+warcs.video().write.csv('/path/to/derivatives/csv/video', header='true')
+
+# Word Processor Files.
+warcs.word_processor().write.csv('/path/to/derivatives/csv/word_processor', header='true')
+```
+
+## Extract Binary Info to CSV (s3)
```

Comment:
Instead of a separate section on S3, just point out what's different? i.e., basically the last line? (and the keys)